packages/runtime/container-runtime/src/summary/summarizerTypes.ts

```diff
@@ -141,6 +141,8 @@ export interface ISubmitSummaryOptions extends ISummarizeOptions {
 	readonly cancellationToken: ISummaryCancellationToken;
 	/** Summarization may be attempted multiple times. This tells whether this is the final summarization attempt. */
 	readonly finalAttempt?: boolean;
+	/** The sequence number of the latest summary used to validate if summary state is correct before summarizing */
+	readonly latestSummaryRefSeqNum: number;
```

Comment:
Does this info make sense to have on this exported (alpha) API? Do all users who would pass this in know how to get this info?  e.g. `ContainerRuntime.submitSummary`.

Ok looking at usages, I'm gathering that's function is the only reason it has to be exported, and that function really should be internal.  So, I guess chalk this up to tech debt as something we should try to figure out how to trim.

Right?

Reply:
I hope no one is really trying to call this api other than us because they could very easily break things. It is really easy to go wrong submitting a summary. You need to know to refresh the latest ack on the container runtime before even submitting a summary. Otherwise, this api won't even work, or anyone else would likely burn themselves in the form of data loss or corruption. It's unpredictable behavior.